src/plugins/orangehrmCorePlugin/Api/V2/RequestParams.php

```diff
@@ -228,12 +230,29 @@ public function getAttachmentOrNull(string $type, string $key, ?array $default =
     {
         $attachment = $this->$type->get($key, $default);
         if (isset($attachment['name']) && isset($attachment['type']) && isset($attachment['base64']) && isset($attachment['size'])) {
-            return new Base64Attachment(
+            $attachment = new Base64Attachment(
                 $attachment['name'],
                 $attachment['type'],
                 $attachment['base64'],
                 $attachment['size']
             );
+
+            // Check for SVG and sanitize
+            if ($attachment->getFileType() === 'image/svg+xml') {
+                $sanitizer = new Sanitizer();
+                try {
+                    $sanitizedContent = $sanitizer->sanitizeSvg($attachment->getContent());
+
+                    $attachment->setContent($sanitizedContent);
+                    $attachment->setSize(strlen($sanitizedContent));
```

Comment:
```suggestion
                    $attachment->setSize($this->getTextHelper()->strLength($sanitizedContent, '8bit'));
```
Check;
`src/plugins/orangehrmCorePlugin/Api/V2/Validator/Rules/Base64Attachment.php`
inside the helper we used `mb_strlen` to handle unicode. have a look

Reply:
changed to use `TextHelper`